### DISCUSSION TITLE
Music Streaming

### DISCUSSION THREAD

**Discussion by Maxxedison** [MissingCore/Music#111] [Ideas]
I know this is must be a huge difficulty for the devoloping team but a streaming service whould be awesome. I just downloaded the app today and i love it.

**Comment by cyanChill** [comment]
Thanks for enjoying the app. As for your request for a streaming service, there's no plans for that as I plan on keeping this strictly a local music player. Supporting streaming services introduces cost, complexity, and the issue of legality.
